pkg/generated/contract/DummyTokenTelemetry.go

```diff
@@ -0,0 +1,196 @@
+// Code generated - DO NOT EDIT.
+// This file is a generated binding and any manual changes will be lost.
+
+package contract
```

Comment:
wait so this file is here bc you were trying to get the deployment to work right?

Reply:
yeah, should be in the `contractutils.go` right now.  will remove if we go with some other method of testing.